### ISSUE TITLE
Dokany uninstaller leaves system files laying around

### ISSUE THREAD

**Issue by gh223682** [dokan-dev/dokany#246]
# Environment
 * Windows version: 10
 * Processor architecture: amd64 
 * Dokany version: 1.0.0-RC3
 * Library type (Dokany/FUSE): Dokany/FUSE
 
# Description
Dokany uninstaller leaves dokan* system files laying around in System32. These files should be removed during the uninstall process. This prevents installation of previous versions of Dokany.

Please see related closed issues #94 and #142.

**Comment by Liryna** [comment]
hello @gh223682 ,

The issues that you linking are not related since it is about the old installer of 0.8.0.

Did you reboot after the uninstall ? Dokan System files are only removed after the reboot.

**Comment by Liryna** [comment]
I am unable to reproduce with RC3.
After reboot, sys / prog folder is well cleaned

**Comment by gh223682** [comment]
@Liryna I've been debugging EncFS4Win issues, and either:

1) Installing Dokan redistributed with the EncFS4Win 1.10.0-RC4 installer (which turns out to be Dokan 1.0.0-RC2, sorry for the error);
2) Installing Dokan separately using the standalone Dokan installer; or
3) Installing Dokan over Dokan installed by the EncFS4Win installer.

It doesn't happen every time I uninstall, but I've experienced this problem at least three separate times. I'm pretty sure I rebooted after each uninstall of Dokan.
